internal/resources/aws/elasticache_cluster.go

```diff
@@ -104,3 +121,56 @@ func (r *ElastiCacheCluster) backupStorageCostComponent() *schema.CostComponent
 		},
 	}
 }
+
+// ElastiCache implementation of reservationResolver
```

Comment:
The convention for `godoc` is to have the name of the type as the first word in the comment. E.g. `elasticacheReservationResolver`. Read more here: https://go.dev/blog/godoc. Can we update all the comments on these types to make sure they're in line with this? Plus could we actually make these a bit more useful comments? I don't see a comment of `implementation of x type` as very useful for other engineers. Maybe remove comments here and just have a more in depth comment over each of the  ` PriceFilter` methods, explaining why the mapping for terms e.t.c.

Reply:
Let me know if them are clear enough 🙏